src/features/SendAssets/helpers.ts

```diff
@@ -126,24 +125,34 @@ export const parseTransactionQueryParams = (queryParams: any) => (
   accounts: StoreAccount[]
 ): ITxConfig | undefined => {
   // if speedup tx does not contain all the necessary parameters to construct a tx config return undefined
-  const i = MANDATORY_TRANSACTION_QUERY_PARAMS.reduce((acc, cv) => {
-    if (queryParams[cv] === undefined) return { ...acc, invalid: true };
-    acc[cv] = queryParams[cv];
-    return acc;
+  const i = SUPPORTED_TRANSACTION_QUERY_PARAMS.reduce((acc, cv) => {
+    if (queryParams[cv] === undefined) return acc;
+    return { ...acc, [cv]: queryParams[cv] };
   }, {} as Record<TxParam, TTxQueryParam>) as IFullTxParam;
```

Comment:
Use generic parameter to specify type for `reduce`.

Reply:
Have simplified this, if you want to go further we need to talk about it 😄